anta/device.py

```diff
@@ -163,6 +250,12 @@ async def collect_commands(self, commands: list[AntaCommand]) -> None:
         """
         await asyncio.gather(*(self.collect(command=command) for command in commands))
 
+    @staticmethod
+    @abstractmethod
+    def is_supported(command: AntaCommand) -> bool:
```

Comment:
so the call would be `device.is_supported(command)` maybe let's change the name in `device.supports_command(command)` ?